### PR TITLE
feat(pager): improve accessibility - INNO-605

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

I had to update the component to make it follow latest guidelines.
Accessibility has been improved too.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [x] `package.json` is up-to-date and `@ec-europa/ecl-base` is part of the dependencies
- [x] I have checked the dependencies
- [x] I have given the fractal status “ready” to my component
- [x] I have declared `@define mycomponent` in the SCSS file
- [x] I have specified `margin: 0;` on the CSS component
- [x] I have provided tests
- [x] I follow the naming guidelines
- [x] the component supports composition
- [x] there are no hardcoded strings (all content come from the context)
- [x] I have filled the README.md file (at least a few lines)
- [x] My component is listed in the root README
- [x] My PR has the right label(s)

### CODE REVIEW

File: framework/components/ecl-pagers/ecl-pagers.twig [hunk 1]
```diff
@@ -1,67 +1,118 @@
...
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-pager' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_class is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_Class %}
```
Comment (kalinchernev) [inline framework/components/ecl-pagers/ecl-pagers.twig]: `extra_Class` => `extra_class`

**Review by kalinchernev — COMMENTED** [review]
Looks and works ok. Only a small naming in 1 variable.

**Comment by emeryro** [comment]
Typo in variable has been fixed

**Comment by yhuard** [comment]
Will check :)

File: framework/components/ecl-pagers/ecl-pagers.scss [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 .ecl-pager__wrapper {
   border-top: 2px solid #000;
+  margin: 0;
```
Comment (yhuard) [inline framework/components/ecl-pagers/ecl-pagers.scss]: 🎉

**Review by yhuard — APPROVED** [review]
Better now!

**Pull request merged**